database/rrdhost.c

```diff
@@ -313,6 +310,22 @@ RRDHOST *rrdhost_create(const char *hostname,
         if (unlikely(rc))
             error_report("Failed to store machine GUID to the database");
         sql_load_node_id(host);
+        if (host->health_enabled) {
+            if (!file_is_migrated(host->health_log_filename)) {
+                int rc = sql_create_health_log_table(host);
+                if (unlikely(rc)) {
+                    error_report("Failed to create health log table in the database");
+                    health_alarm_log_load(host);
+                    health_alarm_log_open(host);
+                }
+                else {
+                    health_alarm_log_load(host);
+                    add_migrated_file(host->health_log_filename, 0);
+                }
+            } else {
+                sql_health_alarm_log_load(host);
```

Comment:
Do you think we should add a check here if the table really exists, and if not re-create it, if it has been deleted by the user? Now, if the user has deleted the health log table, the only re-create will happen if he deletes also the entry in metadata_migration

Reply:
Ok, added in https://github.com/netdata/netdata/pull/11399/commits/97ed0ef80e4009f9f62b33bdeb294c51ff8198e0

Doesn't check return, not sure if it makes sense at that point....